src/components/UTIL_PageMessages.component

```diff
@@ -15,7 +15,7 @@
         </style>
     </apex:outputText>
 
-    <div class="slds" id="page_messages">
```

Comment:
since you are changing one of our shared components, did you test all the other pages to make sure this is OK?  I don't recall if those others are including the slds 203 resource or using another version.